evap/staff/tests/test_forms.py

```diff
@@ -430,3 +431,51 @@ def test_contributor_course_form_date_validation(self):
 
         # staff: but start date must be < end date
         self.helper_date_validation(CourseForm, "02/1/1999", "02/1/1998", False)
+
+
+class TestCourseOperationForms(WebTest):
+    fixtures = ['minimal_test_data']
```

Comment:
eventually i want to remove this file as well. it would be cool if you could generate the relevant data inside this class (or inside the individual tests if possible), although that might be a bit of work.